src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -22,29 +22,30 @@ public class FormulaValueJSON
         /// <summary>
         /// Convenience method to create a value from a json representation. 
         /// </summary>
-        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null)
+        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false)
```

Comment:
@LucGenetier  - this extra parameter here is a binary breaking change, but the api compat tool didn't flag it. 
Is it scanning this dll?